app/contracts/Migrations.json

```diff
@@ -0,0 +1,1385 @@
+{
```

Comment:
Probably good to remove these jsons as they are automatically created.

Reply:
I think we should keep them as gitignoring these JSON files will cause more trouble in the future when after deploying them